build-scripts/hack/util.py

```diff
@@ -48,3 +48,9 @@ def read_file(path: Path) -> str:
 
 def read_url(url: str) -> str:
     return urlopen(url).read().decode().strip()
+
+def helm_pull(chart_name: str, repo_url: str, version: str, destination: Path) -> None:
+    parse_output(["helm", "repo", "add", chart_name, repo_url])
+    parse_output(["helm", "pull", f"{repo_url}/{chart_name}", "--version", version, "--destination", destination])
+
+    LOG.info("Pulled helm repository %s @ %s as %s to %s", repo_url, version, chart_name, destination)
```

Comment:
newline at EOF

Reply:
also, message should read like `LOG.info("Pulled helm chart %s @ %s from %s at %s", chart_name, version, repo_url destination)`